modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSView.tsx

```diff
@@ -54,15 +48,13 @@ let NAVIGATION_BAR_ITEMS = [
 	},
 ];
 
-if (Liferay.FeatureFlags['LPD-10735']) {
-	NAVIGATION_BAR_ITEMS = [
-		...NAVIGATION_BAR_ITEMS,
-		{
-			Component: Settings,
-			label: Liferay.Language.get('settings'),
-		},
-	];
-}
+NAVIGATION_BAR_ITEMS = [
+	...NAVIGATION_BAR_ITEMS,
+	{
```

Comment:
We can move this option directly into the `NAVIGATION_BAR_ITEMS` array

Reply:
Done!